dashboard/src/app/workspaces/list-workspaces/list-workspaces.controller.js

```diff
@@ -107,8 +123,104 @@ export class ListWorkspacesCtrl {
     this.$q.all(promises).then(() => {
       this.isInfoLoading = false;
     }, (error) => {
-      console.log(error);
       this.isInfoLoading = false;
     });
   }
+
+  /**
+   * Delete all selected workspaces
+   */
+  deleteSelectedWorkspaces() {
```

Comment:
is that code could be simplified by subcalls ? looks like we have a lot nested blocks